### PR TITLE
[14.0][MIG] hr_contract_currency

### PR DESCRIPTION
**Pull request by aphon61bank**
Change track_visibility are tracking in hr_contract.py , Ready for review.

### CODE REVIEW

File: hr_contract_currency/models/hr_contract.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+from odoo import api, fields, models
+
+
+class HrContract(models.Model):
+    _inherit = "hr.contract"
+
+    currency_id = fields.Many2one(
+        "res.currency",
+        related=False,
+        readonly=False,
```
Comment (newtratip) [inline hr_contract_currency/models/hr_contract.py:14]: By default related=False and readonly=False, It is not necessary to add it.

**Review by newtratip — COMMENTED** [review]
Some review.

**Review by newtratip — APPROVED** [review]
:+1: LGTM Code review and functional review with runbot

**Comment by aphon61bank** [comment]
@newtratip  I have fixed it, Thank you.

File: hr_contract_currency/models/hr_contract.py (hunk 1 above)
Comment (alexey-pelykh) [reply hr_contract_currency/models/hr_contract.py:14]: It's an override, so necessary: https://github.com/odoo/odoo/blob/8f21148e1a8eaae3bc09ec54377becb2361e7834/addons/hr_contract/models/hr_contract.py#L79

File: hr_contract_currency/models/hr_contract.py [hunk 2]
```diff
@@ -10,8 +10,6 @@ class HrContract(models.Model):
 
     currency_id = fields.Many2one(
         "res.currency",
-        related=False,
```
Comment (alexey-pelykh) [inline hr_contract_currency/models/hr_contract.py]: This change is not correct

**Review by alexey-pelykh — CHANGES_REQUESTED** [review]

File: hr_contract_currency/models/hr_contract.py (hunk 1 above)
Comment (newtratip) [reply hr_contract_currency/models/hr_contract.py:14]: OK, Sorry for my mistake @alexey-pelykh

@aphon61bank Please back your code before.

File: hr_contract_currency/models/hr_contract.py (hunk 2 above)
Comment (aphon61bank) [reply hr_contract_currency/models/hr_contract.py]: @alexey-pelykh  I have fixed it, Thank you.

File: hr_contract_currency/models/hr_contract.py (hunk 1 above)
Comment (aphon61bank) [reply hr_contract_currency/models/hr_contract.py:14]: I have fixed it, Thank you.

**Comment by aphon61bank** [comment]
@alexey-pelykh help for review again please, Thank you.

**Comment by newtratip** [comment]
@alexey-pelykh, I squash 2 commits into 1 commit, Could you please review again ?

**Review by alexey-pelykh — APPROVED** [review]
Code LGTM

**Pull request merged**